members/WAL000001.yaml

```diff
@@ -53,6 +53,12 @@ contact_form:
           selector: "#Comment"
           value: $MESSAGE
           required: true
+    - javascript:
+        - name: phone
+          selectors: [ "#PhoneNumber"]
+          values: [ "$PHONE" ]
+          commands: [ "elements[0].value = values[0].replace(/\D/g,'').length > 10 ? values[0].replace(/\D/g,'').substr(1) : values[0].replace(/\D/g,'')" ]
```

Comment:
this is always going to remove the first number. is that ok?

Reply:
I believe so, as we want the phone number length to be 10 digits, which is what these target forms' accept plus is the phone number length for numbers in the US

Reply:
ok